### PR TITLE
Docs: adds aliases to fix 404s

### PR DESCRIPTION
**Pull request by brendamuir**
adds two aliases to fix 404 errors.

### CODE REVIEW

File: docs/sources/alerting/_index.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 aliases:
   - about-alerting/
```
Comment (jdbaldry) [inline docs/sources/alerting/_index.md:3]: nit: I propose this to be explicit and consistent with others.
```suggestion
  - ./about-alerting/
```

File: docs/sources/alerting/_index.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
...
   - unified-alerting/alerting/
```
Comment (jdbaldry) [inline docs/sources/alerting/_index.md:4]: nit: I propose this to be explicit and consistent with others.
```suggestion
  - ./unified-alerting/alerting/
```

**Review by jdbaldry — APPROVED** [review]
I confirmed both redirects work as intended to fix the 404s at:
https://grafana.com/docs/grafana/latest/silences/ and https://grafana.com/docs/grafana/latest/alerting/unified-alerting/

File: docs/sources/alerting/_index.md (hunk 2 above)
Comment (brendamuir) [reply docs/sources/alerting/_index.md]: thank you! will do

**Pull request merged**